src/test/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/health/DownStreamHealthIndicatorTest.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.reform.iacaseapi.infrastructure.health;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.iacaseapi.infrastructure.config.HealthCheckConfiguration;
+
+@RunWith(MockitoJUnitRunner.class)
+@SuppressWarnings("unchecked")
+public class DownStreamHealthIndicatorTest {
```

Comment:
This unit test needs more tests.  E.g. you are doing a lot of stuff in the constructor which is not tested at all.  E.g. what if a null list of services is retrieved?

Reply:
You may also consider checking that you are calling the constructor the correct number of times.  You can do that using PowerMockito.